scripts/stac/imagery/collection.py

```diff
@@ -163,7 +163,7 @@ def add_item(self, item: dict[Any, Any]) -> None:
                 Link(
                     path=item_self_link["href"],
                     rel=Relation.ITEM,
-                    media_type=StacMediaType.JSON,
+                    media_type=StacMediaType.GEOJSON,
```

Comment:
I think there may be other places this needs to change as well, such as https://github.com/linz/argo-tasks/blob/f1b559c79aa7ab5a942470354ccf1761eef6d346/src/commands/copy/copy-worker.ts#L37

Reply:
Are those changes required for this PR to go through?

Reply:
After inspection the link above could be an issue but that only applies to unknown content types, which these STAC files should not be included in. Items are currently being saved to S3 and copied as `application/geo+json` .
![image](https://github.com/user-attachments/assets/30b96184-579e-4d82-be66-8bba3a902470)